samples/containers/hello-back-button/src/main/java/com/squareup/sample/hellobackbutton/AreYouSureWorkflow.kt

```diff
@@ -25,23 +10,27 @@ import com.squareup.workflow1.Snapshot
 import com.squareup.workflow1.StatefulWorkflow
 import com.squareup.workflow1.WorkflowAction.Companion.noAction
 import com.squareup.workflow1.action
+import com.squareup.workflow1.ui.AlertModalRendering
+import com.squareup.workflow1.ui.AlertModalRendering.Button.NEGATIVE
+import com.squareup.workflow1.ui.AlertModalRendering.Button.POSITIVE
+import com.squareup.workflow1.ui.AlertModalRendering.Event.ButtonClicked
+import com.squareup.workflow1.ui.AlertModalRendering.Event.Canceled
+import com.squareup.workflow1.ui.ModalContainerViewRendering
 import com.squareup.workflow1.ui.WorkflowUiExperimentalApi
-import com.squareup.workflow1.ui.modal.AlertContainerScreen
-import com.squareup.workflow1.ui.modal.AlertScreen
-import com.squareup.workflow1.ui.modal.AlertScreen.Button.NEGATIVE
-import com.squareup.workflow1.ui.modal.AlertScreen.Button.POSITIVE
-import com.squareup.workflow1.ui.modal.AlertScreen.Event.ButtonClicked
-import com.squareup.workflow1.ui.modal.AlertScreen.Event.Canceled
 import com.squareup.workflow1.ui.toParcelable
 import com.squareup.workflow1.ui.toSnapshot
 import kotlinx.android.parcel.Parcelize
 
+@OptIn(WorkflowUiExperimentalApi::class)
+private typealias Rendering = ModalContainerViewRendering<*, AlertModalRendering>
```

Comment:
Note that this one is private -- it was strictly a convenience to avoid some copy / paste, and arguably a bad choice for the reaons you mentioned. Our trend has been to use fewer and fewer type aliases, especially public ones.

Some of the lingering ones are leftovers from before we had the more concise `action {}` lambdas.